### PR TITLE
#307 - NodeHasPkgCount fully implemented

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #307 I've finished `NodeHasPkgCount` implementation and used it in some test. Added todo to continue.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#308](https://github.com/artipie/rpm-adapter/issues/308) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/92458806-f36d-4a15-82f6-16fe2fe6a597); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java [hunk 1]
```diff
@@ -114,22 +105,13 @@ void doesNotMatchWhenPackageAttributeDiffers() throws FileNotFoundException {
     }
 
     @Test
-    @Disabled
     void describesCorrectlyWhenPackageAttributeDiffers() throws FileNotFoundException {
```
Comment (Vatavuk) [inline src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java:108]: @olenagerasimova this test is failing

File: src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java [hunk 2]
```diff
@@ -82,24 +79,18 @@ void doesNotMatchWhenPackagesAmountDiffers() throws FileNotFoundException {
...
-        new NodeHasPkgCount(10, XmlPackage.PRIMARY.tag(), desc).matches(
-            new XMLDocument(
-                Paths.get(NodeHasPkgCountTest.PRIMARY)
-            )
-        );
         MatcherAssert.assertThat(
-            desc.toString(),
-            new IsEqual<>(
-                "2 packages found, `packages` attribute value is 2, expected 10 package count"
+            new NodeHasPkgCount(10, XmlPackage.PRIMARY.tag()),
```
Comment (Vatavuk) [inline src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java:84]: @olenagerasimova this test is failing

File: src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java (hunk 1 above)
Comment (olenagerasimova) [reply src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java:108]: @Vatavuk I can see) It fails on windows only which I do not have, I'll figure it out a bit later

**Comment by olenagerasimova** [comment]
@Vatavuk I've disabled failing test on windows and added todo to figure the problem out

**Comment by 0crat** [comment]
Job was finished in 41 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks!

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**